### PR TITLE
ESM for integrating with other builds/apps; use of HGC.libraries.PIXI

### PR DESCRIPTION
**Pull request by alexkb0009**
Corresponding fourfront commit - https://github.com/4dn-dcic/fourfront/commit/89dd3941271271640bd0beee0dd88a7af43ec8f6

There's another **temporary** branch **[es-modules-es](https://github.com/higlass/higlass-multivec/tree/es-modules-es)** which contains these edits _plus_ the built "es" directory. Is being used temporarily by 4DN until 0.2.2 release.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,9 +1,6 @@
 import register from 'higlass-register';
 
-import StackedBarTrack from './scripts/StackedBarTrack';
-import ScaledStackedBarTrack from './scripts/ScaledStackedBarTrack';
-import BasicMultipleLineChart from './scripts/BasicMultipleLineChart';
-import BasicMultipleBarChart from './scripts/BasicMultipleBarChart';
+import { StackedBarTrack, ScaledStackedBarTrack, BasicMultipleLineChart, BasicMultipleBarChart } from './scripts';
```
Comment (alexkb0009) [inline src/index.js]: Was changed for testing; can swap back.

File: src/scripts/index.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import StackedBarTrack from './StackedBarTrack';
+import ScaledStackedBarTrack from './ScaledStackedBarTrack';
+import BasicMultipleLineChart from './BasicMultipleLineChart';
+import BasicMultipleBarChart from './BasicMultipleBarChart';
+
```
Comment (alexkb0009) [inline src/scripts/index.js]: This index file serves no real purpose aside from testing & possibly allowing `import { ScaledStackedBarTrack, StackedBarTrack } from 'higlass-multivec';` re: `"module" : "./es"` in package.json (haven't tested this from 4dn-side yet actually.. (vs `..from 'higlass-multivec/es/<filename>';`))

File: README.md [hunk 3]
```diff
@@ -54,6 +54,62 @@ Configure the track in your view config; you should be all set from here!
...
+import { StackedBarTrack } from 'higlass-multivec';
+```
+
+Then StackedBarTrack would automatically be imported from the `./es` directory (set via package.json's `"module"` value). Can also import component(s) directly, especially if only need to use one or two, e.g.:
+
+```javascript
+import { StackedBarTrack } from 'higlass-multivec/es/StackedBarTrack';
+```
+
+To use this then along with HiGlassComponent, assuming are developing with React either in an environment where `window` is available (client-side), is via the following:
```
Comment (pkerpedjiev) [inline README.md]:
```suggestion
To use this then along with HiGlassComponent, assuming you are developing with React either in an environment where `window` is available (client-side), is via the following:
```

**Comment by pkerpedjiev** [comment]
Hey Alex, this looks good! Thanks for submitting it.

Would you mind merging with the latest master branch to make sure the tests pass?

**Comment by alexkb0009** [comment]
Getting Travis build failure; unsure but figuring is due to change of `PIXI`->`HGC.libraries.PIXI`, idk.

I feel like it might be worth trying to add PIXI to package.json peerDependencies, add "import PIXI" statements, and then keep using as `PIXI` since is already defined as an external in `webpack.config.js` so existing usages of it (<script> tags to import PIXI, then HGC, then HGC MV - I think (?)) should continue to work as-is.

**Comment by pkerpedjiev** [comment]
You might be able to get away with just adding it as a peer dependency and making sure that the path to is correct in `karma.conf.js`.

When I try to run the tests locally, I get this:

```
28 10 2019 20:54:43.286:WARN [watcher]: Pattern "/Users/pete/projects/higlass-multivec/node_modules/pixi.js/dist/pixi.js" does not match any file.
```

**Comment by alexander-veit** [comment]
Tests are passing now. Is this still ok with you @pkerpedjiev ?

**Pull request merged**